ros1-bridge-ws/docker/Dockerfile

```diff
@@ -0,0 +1,123 @@
+# - https://docs.ros.org/en/humble/Installation/Alternatives/Ubuntu-Development-Setup.html
+################################################################################################
+
+FROM base AS build
+
+# Install packages
+RUN apt update && \
+    apt install -y \
+    curl \
+    git \
+    locales \
+    software-properties-common \
+    lsb-release
+
+# Set locale
+RUN locale-gen en_US en_US.UTF-8 && \
+    update-locale LC_ALL=en_US.UTF-8 LANG=en_US.UTF-8
+
+# Add the ROS 2 apt repository
+RUN add-apt-repository universe && \
+    curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg && \
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null
+
+# Install development tools and ROS tools
+RUN apt update && \
+    apt install -y \
+    build-essential \
+    cmake \
+    python3-flake8 \
+    python3-flake8-blind-except \
+    python3-flake8-builtins \
+    python3-flake8-class-newline \
+    python3-flake8-comprehensions \
+    python3-flake8-deprecated \
+    python3-flake8-docstrings \
+    python3-flake8-import-order \
+    python3-flake8-quotes \
+    python3-pip \
+    python3-pytest \
+    python3-pytest-cov \
+    python3-pytest-repeat \
+    python3-pytest-rerunfailures \
+    python3-rosdep \
+    python3-setuptools \
+    ros-dev-tools
+
+# Create ros2_humble workspace
+RUN mkdir -p /ros2_humble/src
+WORKDIR /ros2_humble
+
+# Install colcon from PyPI, rather than apt packages
+RUN pip3 install -U colcon-common-extensions vcstool
+
+# clone all repos
+RUN vcs import --input https://raw.githubusercontent.com/ros2/ros2/humble/ros2.repos src
+
+# Install dependencies
+RUN rosdep init && \
+    rosdep update && \
+    rosdep install --from-paths src --ignore-src -y --rosdistro humble --skip-keys "fastcdr rti-connext-dds-6.0.1 urdfdom_headers"
+
+# Build the code in the workspace
+RUN colcon build
+
+# Delete packages.ros.org from sources.list and remove conflicts packages
+RUN rm -rf /etc/apt/sources.list.d/ros2.list && \
+    apt remove -y \
+    python3-catkin-pkg \
+    python3-catkin-pkg-modules
+
+# Install ROS 1 from Ubuntu packages
+RUN apt update && \
+    apt install -y ros-core-dev
+
+# Build ros1_bridge from source
+RUN git clone https://github.com/ros2/ros1_bridge src/ros1_bridge
+RUN source install/local_setup.bash && \
+    colcon build --packages-select ros1_bridge --cmake-force-configure
+
+################################################################################################
+# Build release
+# Reference :
+# - https://docs.docker.com/build/building/multi-stage/
+################################################################################################
+
+FROM base AS release
```

Comment:
Since we now include a `debug/Dockerfile`, the `FROM base AS build` part can be removed, and use change `FROM base AS release` to reuse the debugging image to avoid code duplication (and save building time when building both images).

See: https://docs.docker.com/build/building/multi-stage/#use-an-external-image-as-a-stage